.gitignore

```diff
@@ -46,3 +46,6 @@ site/
 !m3db.io/**/vendor
 # Automatically populated from asset sources
 m3db.io/openapi
+
+# GitHub API token
+github_token
```

Comment:
do you need this change once you delete the file?

Reply:
This is where goreleaser expects it to be so might save someone else the trouble

Reply:
meh separate PRs mate =P

Reply:
fiiiiiine haha